src/utils/backend.js

```diff
@@ -1,5 +1,31 @@
 import { BACKEND_URL } from '../config/constants';
-import { wrapTry } from './util';
+// eslint-disable-next-line import/no-cycle
+import store from '../store';
+
+const wrapTry = async (promise) => {
```

Comment:
Moved this method from utils to don't make the whole utils to depend on the store.